javascript/javascript1/week2/Weatherwear.js

```diff
@@ -1,8 +1,10 @@
-function WeatherDressCode(temperature) {
+const WeatherDressCode = temperature(18); 
+function WeatherDressCode (tempearature){
```

Comment:
You are using one constant variable's name as a new function, so it must be giving you errors right? You can't do that.